daemon/api.go

```diff
@@ -734,6 +734,7 @@ type snapInstruction struct {
 	Channel          string        `json:"channel"`
 	Revision         snap.Revision `json:"revision"`
 	CohortKey        string        `json:"cohort-key"`
+	LeaveCohort      bool          `json:"leave-cohort"`
```

Comment:
Should that be just equivalent to setting cohort key to nothing?

Reply:
internally that is what happens. From the outside, we've been wary of distinguishing a key missing from a key having a default value (IOW in our REST API, `{}` and `{"foo": ""}` are equivalent when `"foo"` is defined to be a string).

That's what
>It's a flag in daemon's
snapInstruction (and snapstate.RevisionOptions) just to not have to
fiddle with empty-string-means-no-change vs
empty-string-means-change-to-nothing.

tried to signify

Reply:
Aha, this makes sense, thank you!